src/main/java/me/sat7/dynamicshop/transactions/Calc.java

```diff
@@ -65,11 +65,9 @@ public static double calcTotalCost(String shopName, String idx, int amount)
             value = ShopUtil.ccShop.get().getDouble(shopName+"." + idx + ".value");
         }
 
-        if(median <= 0 || tempStock <= 0)
-        {
+        if (median < 0 || tempStock < 0) {
```

Comment:
Median and stock must always be 1 or greater.
However, in certain situations, it may be zero.
- It can be 0 if the user directly edits the yml file.
- It can be 0 if the user modifies with a command.

Modifications are required in one of two ways:
 A) Modify the other parts of the code so that it never becomes zero in any case.
 B) Leave this as it is.

Reply:
Please note that player can't buy last item. (blocked by code)